spec/reek/smells/feature_envy_spec.rb

```diff
@@ -32,62 +32,69 @@ def golf(hotel)
       end
     EOS
 
-    expect(src).to reek_of(:FeatureEnvy,
-                           lines: [3, 3],
-                           name:  'charlie')
-    expect(src).to reek_of(:FeatureEnvy,
-                           lines: [7, 7],
-                           name:  'hotel')
+    expect(src).
+      to reek_of(:FeatureEnvy, lines: [3, 3], name:  'charlie').
+      and reek_of(:FeatureEnvy, lines: [7, 7], name:  'hotel')
   end
 
   it 'does not report use of self' do
-    expect('def alfa; self.to_s + self.to_i; end').not_to reek_of(:FeatureEnvy)
+    src = 'def alfa; self.to_s + self.to_i; end'
```

Comment:
Yeah, good call here and below. I thought of doing this as well while working on #1056 but then decided against it to not make the PR bigger than it already was.